handler/src/main/java/io/netty/handler/ssl/ReferenceCountedOpenSslContext.java

```diff
@@ -260,9 +260,10 @@ protected void deallocate() {
             int options = SSLContext.getOptions(ctx) |
                           SSL.SSL_OP_NO_SSLv2 |
                           SSL.SSL_OP_NO_SSLv3 |
-                          // Disable TLSv1.3 by default for now. Even if TLSv1.3 is not supported this will
-                          // work fine as in this case SSL_OP_NO_TLSv1_3 will be 0.
-                          SSL.SSL_OP_NO_TLSv1_3 |
+                          // Disable TLSv1.3 by default if the JDK does not support TLSv13 as well.
+                          // Even if TLSv1.3 is not supported this will work fine as in this case SSL_OP_NO_TLSv1_3
+                          // will be 0.
+                          (SslUtils.isTLSv13EnabledByJDK() ? 0 : SSL.SSL_OP_NO_TLSv1_3) |
```

Comment:
Sorry, could you please elaborate? Why enable only TLSv3 by default on the OpenSSL/BoringSSL side when the JDK one supports it? How are those 2 sides related?

Reply:
@slandelle I just thought it may be a good thing to make thins "consistent".

Reply:
I beg to differ.
IMHO, BoringSSL is the most straightforward way for Netty users to get TLSv1.3 support when they are stuck on Java 8 and can't/don't want to go with Oracle builds or patch their JVM with OpenJSSE.

The OpenSSL/BoringSSL side should enable TLSv3 by default as soon as the OpenSSL/BoringSSL version supports it, which is known to be true for current version (2.0.31.Final) of netty-tcnative-boringssl-static.

Reply:
@slandelle after thinking more about it guess you are right. Let me fix this